src/components/Destination.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import { sendDestination } from "../modules/destination";
+
+const SendDestination = () => {
+  let [message, setMessage] = useState("")
+
+  const submitPlace = async (e) => {
+    e.preventDefault();
+    const response = await sendDestination(
```

Comment:
why are we sending off the location here but not when the user picks a location on the map?

Reply:
We've had a bit of a discussion, and this is going to be re worked a bit